recipes/skmer/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "v1.0.0" %}
```

Comment:
the version, should probably not start with `v` isn't it?

Reply:
Fixed it! Thanks for the review